githubbot/githubbot/handler.go

```diff
@@ -213,6 +204,16 @@ func (h *Handler) handleListSubscriptions(msg chat1.MsgSummary) (err error) {
 	var res string
 	for repo, f := range features {
 		res += fmt.Sprintf("- *%s* (%s)\n", repo, &f)
+		if f.Commits {
+			branches, err := h.db.GetAllBranchesForRepo(msg.ConvID, repo)
+			if err != nil {
+				return fmt.Errorf("error getting branches for repo: %s", err)
+			}
+
+			for _, branch := range branches {
```

Comment:
this should be consistent with gitlabbot, right now that doesn't show master. Might make more sense to show it there too

Reply:
hm true, would be tricky to get the default branch though unless we required user auth for the list command

Reply:
I'm saying just show master on gitlabbot so the two outputs are more similar, we don't have to denote which branch is default

Reply:
@joshblum oh i just meant that "master" might not be the branch you're subscribed to by default so in some cases it wouldn't make sense to show master here. that's probably a very small set of users though so it might make sense to just put master here